### PR TITLE
ZEPPELIN-3140. Fixed Checkstyle issues in zeppelin-server

### PR DESCRIPTION
**Pull request by HorizonNet**
### What is this PR for?
Fixed Checkstyle issues in zeppelin-server module.

### What type of PR is it?
Improvement

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3140

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by HorizonNet** [comment]
@felixcheung Fixed the review comment. Let's wait for the build results.

File: zeppelin-server/src/main/java/org/apache/zeppelin/realm/ActiveDirectoryGroupRealm.java [hunk 1]
```diff
@@ -152,7 +152,7 @@ private String getSystemPassword() {
           password = new String(credEntry.getCredential());
         }
       } catch (Exception e) {
-        log.debug("ignored exception: " + e.getMessage());
+        log.debug("", e);
```
Comment (felixcheung) [inline zeppelin-server/src/main/java/org/apache/zeppelin/realm/ActiveDirectoryGroupRealm.java]: eh, I meant `log.debug("ignored exception", e);`

Comment (felixcheung) [reply zeppelin-server/src/main/java/org/apache/zeppelin/realm/ActiveDirectoryGroupRealm.java]: or better, `log.debug("ignored error from getting credential entry from provider", e);`
or `log.debug("ignored error from getting credential entry from keystore", e);`

Comment (HorizonNet) [reply zeppelin-server/src/main/java/org/apache/zeppelin/realm/ActiveDirectoryGroupRealm.java]: My fault. I added a new commit addressing your comment.

**Comment by felixcheung** [comment]
merging if no more comment

**Comment by felixcheung** [comment]
weird.. I thought I merged it and resolved the bug yesterday.

**Comment by jongyoul** [comment]
Yes, there seemed to be an inconsistency between apache wis and apache github. It's resolved when I merge one more PR into wis.

**Pull request closed without merging**